server/form-pages/utils/index.ts

```diff
@@ -78,6 +78,17 @@ export const getTaskName = <T>(page: T) => {
   return Reflect.getMetadata('page:task', page)
 }
 
+export const updateData = (page: TasklistPage, assessment: ApprovedPremisesAssessment): ApprovedPremisesAssessment => {
```

Comment:
Would this be better places in the `assessmentUtils.ts` file? or does that make mocking it hard when needed. 
If we are keeping it here can we make it 'updateAssessmentData'?

Reply:
👍